### DISCUSSION TITLE
Authenticated Proxy

### DISCUSSION THREAD

**Discussion by federicomameli** [cefsharp/CefSharp#3554] [Q&A]
Hello,
I am embedding a CefSharp component in windows form application
I need to inherit proxy configuration from the system. This works just fine until the proxy need authentication (user and password). In this case Cef Sharp is not asking user and password like usually browsers do. I tried passing --proxy-auto-detect command line argument with the same result.

**Comment by amaitland** [comment]
There is no default dialog for authentication, you need to implement GetAuthCredentials see https://github.com/cefsharp/CefSharp/wiki/General-Usage#proxy-authentication for more detail.

**Reply by federicomameli** [reply]
Thanks
